tests/scripts/thread-cert/message.py

```diff
@@ -166,6 +174,22 @@ def assertMleMessageContainsTlv(self, tlv_class_type):
                 break
 
         assert(contains_tlv == True)
+        return tlv
+
+    def assertAssignedRouterQuantity(self, router_quantity):
+        """ To confirm if Leader contains the Route64 TLV with router_quantity assigned Router IDs.
```

Comment:
Does this follow [PEP 257](https://www.python.org/dev/peps/pep-0257/) docstring conventions?

Reply:
I think I have fixed it. Could you please check it again?
I follow [url=https://google.github.io/styleguide/pyguide.html#Comments]Google Python Style Guide[/url] as before the code style.

Reply:
I think I have fixed it. Could you please check it again?
I follow [url=https://google.github.io/styleguide/pyguide.html#Comments]Google Python Style Guide[/url] as before the code style.[a](https://google.github.io/styleguide/pyguide.html)